tests/endtoend_tests/test_endtoend.py

```diff
@@ -724,8 +762,8 @@ def apply(inputs: InputSchema) -> OutputSchema:
     docker_cleanup["images"].append(img_tag)
 
 
-def logging_tesseract_api_string():
-    return dedent(
+def build_logging_tesseract(workdir):
```

Comment:
Drive-by refactor

Reply:
Looks good, only comment I have is why is this not a fixture?

Reply:
Addressed in #318 :)